Code/Source/RobotControl/ControlConfiguration.h

```diff
@@ -19,7 +18,7 @@ namespace ROS2
     //! Configuration for handling of robot control buses.
     //! Used through ROS2RobotControlComponent.
     struct ControlConfiguration
-    {
+    { // TODO - this is two things - steering type and subscription configuration. Divide
```

Comment:
Would it be better to divide this struct now vs. later? in case the changes will cascade too much

Reply:
I added an issue for this since a good solution would also affect several other parts. This is good to do before Manipulation since that would be another subscription and beneficial for reuse.

The issue is here: https://github.com/RobotecAI/o3de-ros2-gem/issues/230.

I removed the comment.